kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyDeploymentDependentResource.java

```diff
@@ -197,6 +202,10 @@ private Container proxyContainer(KafkaProxyContext kafkaProxyContext,
                     .withMountPath(ProxyDeploymentDependentResource.CONFIG_PATH_IN_CONTAINER)
                     .withSubPath(ProxyConfigDependentResource.CONFIG_YAML_KEY)
                 .endVolumeMount()
+                .withNewResources()
+                    .withRequests(DEFAULT_LIMITS)
+                    .withLimits(DEFAULT_LIMITS)
+                .endResources()
```

Comment:
How does this work if users want to deploy with different limits? Aren't we going to force these limits on every pass of the reconciler?

Reply:
Oh is this an order of operations question? In the scope of this PR there is no way for users to add their own limits and that will be fixed elsewhere?